src/opnsense/mvc/app/models/OPNsense/Proxy/Proxy.xml

```diff
@@ -69,6 +69,14 @@
                 <default>0</default>
                 <Required>N</Required>
             </suppressVersion>
+            <VisibleEmail type="TextField">
+                <default>localhost</default>
+                <Required>N</Required>
+            </VisibleEmail>
+            <VisibleHostname type="TextField">
+                <default>admin@localhost</default>
+                <Required>N</Required>
```

Comment:
Maybe add a regexp match here to match only valid hostnames (we don't have a hostname type yet)